ToyBox/classes/Infrastructure/CasterHelpers.cs

```diff
@@ -192,67 +191,18 @@ public static class CasterHelpers {
 
             return known.Count;
```

Comment:
Where did the implementation of this go?

Reply:
Nowhere. Git's detecting the diff in a weird way. All I deleted was the old stuff in the `#else` block along with the `#if true` line.

Reply:
Also, if you're looking at the PR, there are some lines hidden that you need to expand.